src/sass/profile/card.scss

```diff
@@ -73,7 +73,7 @@
         }
     }
 
-    .profile-joindate, .profile-location, profile-website {
+    .profile-joindate, .profile-location, .profile-website {
         color: var(--fg_faded);
         margin: 2px 0;
```

Comment:
With this change the section gets slightly bigger, remedied by this:
```suggestion
        margin: 1px 0;
```

Reply:
done.